--- setup.py.old	2012-10-27 01:29:29.939204548 +0200
+++ setup.py	2012-10-27 01:37:09.927165404 +0200
@@ -32,48 +32,6 @@
         license = "MIT"
     )
 
-    if sys.platform.startswith("linux") and "install" in sys.argv:
-        stdout_handle = os.popen(app_name + " --icons")
-        icons = stdout_handle.read().replace("\n", "")
-        f = open(os.path.join("/", "usr", "share", "applications", app_name + ".desktop"), "w")
-        f.write("""[Desktop Entry]
-Name=Ryouko
-GenericName=Web Browser
-Comment=Simple PyQt4 Web Browser
-
-Icon=""" + os.path.join(icons, "logo.svg") + """
-
-Type=Application
-Categories=Network;
-
-Exec=""" + app_name + """ %F
-StartupNotify=false
-Terminal=false
-MimeType=text/html;text/webviewhtml;text/plain;image/jpeg;image/png;image/bmp;image/x-windows-bmp;image/gif;""")
-        f.close()
-
-elif "install-singleuser" in sys.argv:
-    if os.path.exists(target_dir):
-        shutil.rmtree(target_dir)
-    shutil.copytree(app_lib, target_dir)
-    if not os.path.exists(os.path.join(os.path.expanduser("~"), ".local", "share", "applications", "network")):
-        os.makedirs(os.path.join(os.path.expanduser("~"), ".local", "share", "applications", "network"))
-    f = open(os.path.join(os.path.expanduser("~"), ".local", "share", "applications", "network", app_name + ".desktop"), "w")
-    f.write("""[Desktop Entry]
-Name=Ryouko
-GenericName=Web Browser
-Comment=Simple PyQt4 Web Browser
-
-Icon=""" + os.path.join(os.path.expanduser("~"), app_name, app_name + "_lib", "icons", "logo.svg") + """
-
-Type=Application
-Categories=Network;
-
-Exec=""" + os.path.join(os.path.expanduser("~"), app_name, app_name) + """ %F
-StartupNotify=false
-Terminal=false
-MimeType=text/html;text/webviewhtml;text/plain;image/jpeg;image/png;image/bmp;image/x-windows-bmp;image/gif;""")
-    f.close()
     if os.path.exists(os.path.join(os.path.expanduser("~"), "bin", app_name)):
         os.remove(os.path.join(os.path.expanduser("~"), "bin", app_name))
     elif not os.path.isdir(os.path.join(os.path.expanduser("~"), "bin")):
